scaffolds/scala/src/main/scala/euranova/TheThing.scala

```diff
@@ -0,0 +1,11 @@
+package euranova
+
+class TheThing{
+
+  val isReady = true
+
+  def theProp: String = {
```

Comment:
This is the only method where you use an equals sign behind the method declaration. I would do it consistently throughout the scaffolding.